components/centraldashboard/app/api_contributors.ts

```diff
@@ -94,6 +177,24 @@ export class ContributorAPI {
         return users;
     }
     routes() {return Router()
+        .get(
+            '/exists',
+            async (req: Request, res: Response) => {
+                const response: HasWorkgroupResponse = {
+                    hasAuth: req.user.hasAuth,
+                    user: req.user.username,
+                    hasWorkgroup: false,
+                };
+                if (req.user.hasAuth) {
+                    const workgroup = await ContributorAPI.getWorkgroupInfo(
```

Comment:
See my previous comment about the static method. Unless there is something that I am missing, it seems like it could easily be converted to an instance method.

Reply:
It is also used in api.ts hence is a static method that can be used outside of an instantiation